.pipelines/osm-arc-prod.yaml

```diff
@@ -0,0 +1,19 @@
+jobs: 
+  - job: test_osm_arc_prod
+    pool: staging-pool
+    workspace: 
+      clean: all
+    steps:
+    - template: templates/aks-setup.yaml
+    - template: templates/enable-arc.yaml
+    - template: templates/add-arc-extension.yaml
+      parameters: 
+        imageTag: $(EXTENSION_VERSION)
```

Comment:
How do these parameters get set?

Reply:
EXTENSION_VERSION and EXTENSION_RELEASE_TRAIN get set through the ADO UI, where we will trigger the pipeline for manual runs